docs/docs_environment.yml

```diff
@@ -17,7 +17,7 @@
 # Authors:
 # CIL Developers, listed at: https://github.com/TomographicImaging/CIL/blob/master/NOTICE.txt
 
-name: docs
+name: cil_testing
```

Comment:
should this be here?

Reply:
Thanks @paskino I have added it to my gitignore about 5 times! Any tips?